manifests/cf-manifest/spec/support/manifest_helpers.rb

```diff
@@ -83,6 +83,7 @@ def load_runtime_config
     runtime_config = render([
       File.expand_path("../../../../shared/build_manifest.sh", __FILE__),
       File.expand_path("../../../runtime-config/runtime-config-base.yml", __FILE__),
+      File.expand_path("../../../runtime-config/datadog-addon.yml", __FILE__),
```

Comment:
this is to render runtime config. We have decided to test with this added. Ideally, we'd have matrix of tests where we test for every feature enabled/disabled. But for that, we at least need unified build/render scripts.